README.md

```diff
@@ -49,20 +49,21 @@ Note:
 if you experience any error with the above command, try `az login` first for [Azure Authentication for pulling latest docker images](#azure-authentication-for-pulling-latest-docker-images)
 
 
+3. If ccd-docker already setup and all images/volumes exists and database points to Postgres9 then prior to pulling images run the below commands to delete existing images and volumes.
 
-3. Add Postgres V11 DB settings - **THIS STEP IS ONLY REQUIRED IF YOU NEED TO MIGRATE TO POSTGRES V11**
-- [Postgres v11 database set-up](/PostgresV11-prerequisites.md)
-
-
-
-4. Pull latest Docker images:
```

Comment:
"Pull latest Docker images:" should not be deleted as it relates to compose pull command